lua/wikis/commons/Widget/Infobox/TeamHistory.lua

```diff
@@ -0,0 +1,87 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/Infobox/TeamHistory
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+
+local Info = Lua.import('Module:Info', {loadData = true})
+local Logic = Lua.import('Module:Logic')
+local HtmlWidgets = Lua.import('Module:Widget/Html/All')
+local TeamHistoryAuto = Lua.import('Module:Infobox/Extension/TeamHistoryAuto')
+local Widget = Lua.import('Module:Widget')
+local Widgets = require('Module:Widget/All')
+
+local Big = HtmlWidgets.Big
+local Div = HtmlWidgets.Div
+
+local DEFAULT_MODE = 'manual'
+
+---@alias automatedHistoryMode 'manualPrio'|'cleanup'|'merge'|'automatic'|'manual'
```

Comment:
Could we include a comment with a short description for each mode here? I assume Martin won't be the only one who asks about it and has no time, motivation or knowledge to read the code.

Reply:
good idea, will likely add tomorrow